plugins/dynamic-home-page/src/hooks/useHomePageMountPoints.ts

```diff
@@ -0,0 +1,30 @@
+import { useScalprum } from '@scalprum/react-core';
+
+import { HomePageCardMountPoint } from '../types';
+
+interface ScalprumState {
+  api: {
+    dynamicRootConfig: {
+      mountPoints: {
+        'home.page/card': HomePageCardMountPoint[];
```

Comment:
I'd suggest having this plural for consistency with the other mountpoint names, i.e: `home.page/cards`

Reply:
Changed to plural, also if my feeling is that we should use the singular everywhere ;), but consistency is key.